sonar-core/src/main/java/org/sonar/core/user/GroupMembershipDao.java

```diff
@@ -47,6 +48,16 @@ public GroupMembershipDao(MyBatis mybatis) {
     }
   }
 
+  public int countGroups(GroupMembershipQuery query, Long userId) {
```

Comment:
I would prefer to add the session in parameter. If all methods of all daos were using session, we could remove the dependency on MyBatis from Daos.